jre_emul/tests.mk

```diff
@@ -232,6 +232,9 @@ run-net-tests: link resources $(TEST_BIN)
 run-nio-tests: link resources $(TEST_BIN)
 	@$(TEST_BIN) org.junit.runner.JUnitCore com.google.j2objc.nio.NioTests
 
+run-security-tests: link resources $(TEST_BIN)
+	@$(TEST_BIN) org.junit.runner.JUnitCore com.google.j2objc.security.SecurityTests
+
```

Comment:
Please add this to the initialTests array in junit_tests/JRETestsTableViewController.m.

Reply:
Done